api/tests/mocha/services/generate-xform.spec.js

```diff
@@ -82,16 +86,36 @@ describe('generate-xform service', () => {
 
     it('correctly replaces models with nested "</root>" - #5971', () => runTest('nested-root'));
 
-    it('errors if child process errors', done => {
-      runTest('simple', 'some error')
-        .then(() => done(new Error('expected error to be thrown')))
-        .catch(err => {
-          expect(err.message).to.equal(
-            'Error transforming xml. xsltproc returned code "100", and signal "undefined". ' +
-            'xsltproc stderr output:\nsome error'
-          );
-          done();
-        });
+    it('errors if child process errors', async () => {
+      try {
+        await runTest('simple', 'some error');
+        assert.fail('expected error to be thrown');
+      } catch (err) {
+        expect(err.message).to.equal(
+          'Error transforming xml. xsltproc returned code "100", and signal "undefined". ' +
+          'xsltproc stderr output:\nsome error'
+        );
+      }
+    });
+
+    it('errors if xsltproc command not found', async () => {
+      try {
+        const err = new Error('Command xsltproc not found');
```

Comment:
```suggestion
        const err = new Error('Error: write EPIPE');
```
Here better to specify a different message from the one we added in the reject, so we can check the reject message was passed on.

Reply:
Good point.